bigbluebutton-client/src/org/bigbluebutton/modules/users/views/MediaItemRenderer.mxml

```diff
@@ -199,6 +200,7 @@
 							muteImg.includeInLayout = !rolledOver;
 							muteBtn.visible = rolledOver;
 							muteBtn.includeInLayout = rolledOver;
+							muteBtn.enabled = true;
```

Comment:
It looks like this is the only place you set muteBtn.enabled to something, which means that it will probably not work properly. Try this:
- join two users as moderator
- turn on the microphone on both
- demote one of the users to viewer
- as viewer, try to mute/unmute the moderator
If you can do it, it's not the expected behavior.